.gitignore

```diff
@@ -15,3 +15,8 @@ pkged.go
 .vscode
 vendor/
 benchmark/git-info.env
+
+storage/
+playground/
+pyroscope
+LOCAL_TODO.md
```

Comment:
Can we remove these?

`storage/` and `pyroscope` are taken care of if you use `make` to build and start `pyroscope`:
* `make build`: https://github.com/pyroscope-io/pyroscope/blob/main/Makefile#L35
* `make server`: https://github.com/pyroscope-io/pyroscope/blob/main/Makefile#L59

^ these are also documented in [Developed Guide](https://pyroscope.io/docs/developer-guide)

RE `LOCAL_TODO.md` & `playground` — I'd say let's keep it if these were commonly used conventions, but afaik they're not. So I would suggest you put these in your local .gitignore, e.g. [~/.gitignore](https://git-scm.com/docs/gitignore)

Reply:
Also please set up your editor to add a new line in the end of files, it's a good practice. See [this stackoverflow answer for context](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file).